dbt/models/default/schema.yml

```diff
@@ -31,15 +31,15 @@ models:
             - pin
             - year
           config:
-            error_if: ">280679"
+            error_if: ">280721"
```

Comment:
Bumping these thresholds yet again... I'm going to chat with Billy tomorrow (Wednesday) to see if we can remove these tests.

Reply:
**suggestion:** I recommend you comment these out for now. We can re-add them once we dive more into the tests.